1.20/s2i/bin/assemble

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-set -e
```

Comment:
This has to be set. In case the `assemble` script is failing for some reason, the build should not continue. Even if `cp -Rf /tmp/src/....` is failing.

Reply:
Makes sense. I was also thinking, that with unsetting exit-on-error only here, I can cause confusion, because it is present everywhere else.
I did this, because having to remove `set -x` option from multiple files in order to find out, why the tests are failing makes the debugging situation a bit difficult.
In any way, you are true, that I should remove this change, thx.